packages/loader/src/resource-deserialize/resources/texture2D/TextureDecoder.ts

```
@@ -40,33 +40,48 @@ export class Texture2DDecoder {
       } else {
-        const blob = new window.Blob([imagesData[0]]);
-        const img = new Image();
-        img.onload = () => {
+        let completedCount = 0;
+        const onComplete = () => {
+          completedCount++;
+          if (completedCount >= mipCount) {
+            resolve(texture2D);
+          }
+        };
+        this._loadImageBuffer(imagesData[0]).then((img) => {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)